HISTORY.rst

```diff
@@ -30,6 +30,9 @@ unreleased
   * Modification of how route testing works for testing all route sources. If `az iot hub message-route test` is called
     without specifying a route name or type, all types will be tested rather than only DeviceMessage routes.
 
+  * Addition of new parameters is introduced in `az iot hub configuration create` and
+    `az iot edge deployment create` to let user insert labels and metrics in key-value pair style
+
```

Comment:
This history/changelog entry was not part of 0.18.3, please move it up to this unreleased set

![image](https://user-images.githubusercontent.com/28658112/211424060-aa3d27cc-42ab-4131-a8bd-7a795480b857.png)